_includes/pbc/all/install-features/202304.0/install-the-product-labels-promotions-and-discounts-feature.md

```diff
@@ -0,0 +1,68 @@
+
+
+## Install feature core
+
+Follow the steps below to install the feature core.
+
+### Prerequisites
+
+To start feature integration, overview, and install the necessary features:
+
+| NAME                   | VERSION          |
+|------------------------|------------------|
+| Promotions & Discounts | {{site.version}} |
+| Product Labels         | {{site.version}} |
+| Spryker Core           | {{site.version}} |
+
+### 1) Set up behavior
+
+Set up the following behaviors:
+
+| PLUGIN                                      | SPECIFICATION                                                                      | PREREQUISITES | NAMESPACE                                                                   |
+|---------------------------------------------|------------------------------------------------------------------------------------|---------------|-----------------------------------------------------------------------------|
+| ProductLabelDiscountableItemCollectorPlugin | Collects the cart items with product labels to which a discount should be applied. | None          | Spryker\Zed\ProductLabelDiscountConnector\Communication\Plugin\Collector    |
+| ProductLabelListDecisionRulePlugin          | Defines if a discount can be applied to a cart item with a product label.          | None          | Spryker\Zed\ProductLabelDiscountConnector\Communication\Plugin\DecisionRule |
+
+**src/Pyz/Zed/Discount/DiscountDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Discount;
+use Spryker\Zed\Discount\DiscountDependencyProvider as SprykerDiscountDependencyProvider;
+use Spryker\Zed\ProductLabelDiscountConnector\Communication\Plugin\DecisionRule\ProductLabelListDecisionRulePlugin;
+use Spryker\Zed\ProductLabelDiscountConnector\Communication\Plugin\Collector\ProductLabelDiscountableItemCollectorPlugin;
+
+class DiscountDependencyProvider extends SprykerDiscountDependencyProvider
+{
+    /**
+     * @return \Spryker\Zed\Discount\Dependency\Plugin\DecisionRulePluginInterface[]
+     */
+    protected function getDecisionRulePlugins()
+    {
+        return array_merge(parent::getDecisionRulePlugins(), [
+            new ProductLabelListDecisionRulePlugin(),
+        ]);
+    }
+
+    /**
+     * @return \Spryker\Zed\Discount\Dependency\Plugin\CollectorPluginInterface[]
+     */
+    protected function getCollectorPlugins()
+    {
+        return array_merge(parent::getCollectorPlugins(), [
+            new ProductLabelDiscountableItemCollectorPlugin(),
+        ]);
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that the plugins work correctly:
+
+1. [Create a discount](/docs/pbc/all/discount-management/{{site.version}}/manage-in-the-back-office/create-discounts.html) and define its condition as a query string with a *product-label* field.
+2. Add a product with the defined product label to cart.
+3. The discount should be applied to the cart.
+
+{% endinfo_block %}
```

Comment:
* Also, we have to add the page description document, check the following example: docs/scos/dev/feature-integration-guides/202304.0/install-push-notification-feature.md